templates/default.yaml

```
@@ -10,14 +10,18 @@ STM32L496G_BG96_binary_step:
   0. Download the `Anjay-freertos-client-STM32L496G-BG96.bin` file.
 B_L462_CELL1_binary_step:
   0. Download the `Anjay-freertos-client-B-L462E-CELL1-TYPE1SC.bin` file.
+nRF9160DK_flashing_tool_prerequisit:
```

```suggestion
nRF9160DK_flashing_tool_prerequisite:
```
You can leave the previous version but I only wanted to highlight the proper form of this word